src/events/request/RequestUpdateEventHandler.ts

```diff
@@ -0,0 +1,46 @@
+import { Message, TextChannel } from 'discord.js';
+import * as log4js from 'log4js';
+import EventHandler from '../EventHandler';
+import { RequestsUtil } from '../../util/RequestsUtil';
+import DiscordUtil from '../../util/DiscordUtil';
+
+export default class RequestUpdateEventHandler implements EventHandler<'messageUpdate'> {
```

Comment:
Yay! It's always somewhat bothered me that "update" was just "delete and post anew". This is way better :+1:

Reply:
Thanks! :D